### ISSUE TITLE
Broken out for SVG with PNG fallback (Multiple background-image) 

### ISSUE THREAD

**Issue by xzyfer** [jakubpawlowicz/clean-css#553]
A variant of this issue was originally reported in https://github.com/jakubpawlowicz/clean-css/issues/371. 

IN
```css
foo {
  background: url("foo.png");
  background: none, url("foo.svg");
}
```
OUT
```css
foo{background:0 0,0 0/auto,auto repeat,repeat none,transparent}
```

Switching the position of `none` (as reported in https://github.com/jakubpawlowicz/clean-css/issues/371) fixes the problem.

IN
```css
foo {
  background: url("foo.png");
  background: url("foo.svg"), none;
}
```
OUT
```css
foo{background:url(foo.png);background:url(foo.svg),none}
```

***
I'm using v3.2.5 the following options
```js
{
      processImport: false,
      rebase: false,
      restructuring: false,
}
```

**Comment by jakubpawlowicz** [comment]
Good catch :+1:

**Comment by jakubpawlowicz** [comment]
@xzyfer It's fixed in 3.2.7.

**Comment by xzyfer** [comment]
@jakubpawlowicz thanks!
